west.yml

```diff
@@ -50,7 +50,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: b341c1ffbee9e1694f07a62f55b76b3f4947f603
+      revision: pull/894/head
```

Comment:
Please rebase and update SHA: `2b86c72404d376489a9a37cf6867a8710bdd5401` (previous CI run was green after reruns).